lib/trento/settings/installation_settings.ex

```diff
@@ -13,7 +13,7 @@ defmodule Trento.Settings.InstallationSettings do
   @derive {Jason.Encoder, except: [:__meta__, :__struct__]}
   @primary_key {:id, :binary_id, autogenerate: true}
   schema "settings" do
-    field :eula_accepted, :boolean, source: :installation_settings_eula_accepted
+    # field :eula_accepted, :boolean, source: :installation_settings_eula_accepted
```

Comment:
This line has been commented out to enable `installation_id` to be returned. I would be interested in another way to solve this since `eula_accepted` column seems to have been removed from the table.